core/src/main/java/com/dooapp/fxform/view/skin/FXMLSkin.java

```diff
@@ -40,12 +40,15 @@
 
     private Initializable controller;
 
+	private final FXForm fxForm;
+
     public FXMLSkin(FXForm fxForm, URL url) {
         this(fxForm, url, null);
     }
 
     public FXMLSkin(FXForm fxForm, URL url, Initializable controller) {
         super(fxForm);
+        this.fxForm = fxForm;

```

Comment:
You don't need to save a reference to the fxform here, it's already done in SkinBase. You can access the fxform using `getSkinnable()`